GameUserSettings.ini

```
@@ -36,6 +36,10 @@ EnableCryoSicknessPVE=false
 XPMultiplier=7.5
 DifficultyOffset=1.0
 ServerPVE=true
+AlwaysAllowStructurePickup
+StructurePickupTimeAfterPlacement
+AllowIntegratedSPlusStructures
+StructurePickupHoldDuration
```

I am not expert, but don't they need to be like this?

👇 

```suggestion
AlwaysAllowStructurePickup=true
StructurePickupTimeAfterPlacement=true
AllowIntegratedSPlusStructures=true
StructurePickupHoldDuration=true
```